cypress/fixtures/Certificate/CourseCertificate/credentials.json

```diff
@@ -2,5 +2,13 @@
   "free": {
     "email": "e2e-free-course-certificate@example.com",
     "password": "Abc12345-"
+  },
+  "standard": {
+    "email": "e2e-standard-course-certificate@example.com",
+    "password": "Abc12345-"
+  },
+  "standard_unawarded": {
+    "email": "e2e-standard-course-certificate-unawarded@example.com",
+    "password": "Abc12345-"
```

Comment:
This account does not exist, to make things simpler and the tests more realistic, please just reuse the same standard account, I've just added a pending course. 

![image](https://github.com/bixlabs/CypressAcapediaProfessionals/assets/6330366/f33ddaee-0f6b-47da-b271-ee8e177d6327)


![image](https://github.com/bixlabs/CypressAcapediaProfessionals/assets/6330366/27d3c6f2-92f9-4cb5-a1e7-9eb4af30d890)

```suggestion
```

Reply:
I would prefer to add an account just for the unawarded, that would make things easier actually for the test

Reply:
It's more realistic to have the combination of things, for free we did in that way and it worked, the idea is to have only one credential per module, I did run your tests using the users `standard` and it worked without extra effort.

Reply:
i dont know rick, due that the data has 2 "Review course" and it could grab whatever (the not pending) making the test to fail or at leat a bit flaky, but ok